enterprise-suite/scripts/lbc.py

```diff
@@ -828,5 +901,6 @@ def main(argv):
             check_kubectl()
         debug_dump(args)
 
+
 if __name__ == '__main__':
     main(sys.argv[1:])
```

Comment:
Sorry, the lack of correct formatting (per pep-8) was driving me nuts. So I formatted the file - but probably should have done that in a separate PR.